pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```diff
@@ -3,17 +3,25 @@
 , lib
 }:
 
-ffmpeg_6-full.overrideAttrs (old: rec {
-  pname = "jellyfin-ffmpeg";
+let
   version = "6.0.1-3";
+in
 
-  src = fetchFromGitHub {
+(ffmpeg_6-full.override {
+  inherit version; # Important! This sets the ABI.
+  source = fetchFromGitHub {
     owner = "jellyfin";
     repo = "jellyfin-ffmpeg";
     rev = "v${version}";
     hash = "sha256-UINiXO61nB/AL0HJJy7G7emujakk/mQv81aUioyJz0Y=";
   };
 
+  # Enabled despite the lower ABI
+  withHarfbuzz = true;
+  withAribcaption = true;
```

Comment:
This doesn't work currently, as the enable string gets filtered out for versions below `6.1`

Reply:
Indeed but I diffed the closures and the buildInput was included before. I had assumed it may have backported the patches and done something with it but it's more likely that the buildInput ended up there erroneously.

Reply:
Did some greping through the jellyfin-ffmpeg source code, but did not find any reference to aribcaption and the only reference I found to harfbuzz was a script downloading and compiling it.

<details>
  <summary>Result of  <code class="notranslate">grep -ri harfbuzz</code>:</summary>
  <ul>
    <li>builder/scripts.d/45-harfbuzz.sh:SCRIPT_REPO="https://github.com/harfbuzz/harfbuzz.git"</li>
    <li>builder/scripts.d/45-harfbuzz.sh:    git-mini-clone "$SCRIPT_REPO" "$SCRIPT_COMMIT" harfbuzz</li>
    <li>builder/scripts.d/45-harfbuzz.sh:    cd harfbuzz</li>
    <li>docker-build-win64.sh:# HARFBUZZ</li>
    <li>docker-build-win64.sh:git clone --depth=1 https://github.com/harfbuzz/harfbuzz.git</li>
    <li>docker-build-win64.sh:pushd harfbuzz</li>
  </ul>
</details>

Reply:
The jellyfin-ffmpeg maintainers should get pinged now; please weigh in.